src/libstd/sys/unix/stack_overflow.rs

```diff
@@ -138,11 +139,15 @@ mod imp {
 
     unsafe fn get_stackp() -> *mut libc::c_void {
         let stackp =
-            mmap(ptr::null_mut(), SIGSTKSZ, PROT_READ | PROT_WRITE, MAP_PRIVATE | MAP_ANON, -1, 0);
+            mmap(ptr::null_mut(), SIGSTKSZ + page_size(), PROT_READ | PROT_WRITE, MAP_PRIVATE | MAP_ANON, -1, 0);
```

Comment:
This long line is going to get flagged by `./x.py fmt --check`. You could compute the size separately, or just accept the reflow by running `./x.py fmt` yourself.

Reply:
I was wondering about that. I naively tried a `cargo fmt` and reformatted all of rust, which seemed wrong. Undoubtedly that used a different toolchain with older style opinions. Thanks.